src/platforms/react-native/performance/included-instrumentation.mdx

```diff
@@ -4,8 +4,8 @@ sidebar_order: 10
 description: "Learn what transactions are captured after tracing is enabled."
 ---
 
-<Alert level="Info" title="Beta Support">
-Automatic instrumentation is currently in beta, available on the Sentry React Native SDK version ≥ 2.2.0-beta.0. As a result, the API may change without warning.
+<Alert level="Info" title="Minimum Version">
+To use Automatic instrumentation, you will need at least version 2.2.0 of the Sentry React Native SDK.
```

Comment:
I'd recommend moving this to the include on the set up page `src/includes/performance/enable-tracing/react-native.mdx` similar to what we've done for the other SDKs. It's a more natural to discuss version when you're setting up than when you're learning about what we've included :)  

```suggestion
To use our included instrumentation, you will need at least version 2.2.0 of the Sentry React Native SDK.
```